samples/docker_aks/Makefile

```diff
@@ -19,7 +19,7 @@ $(APPDIR): hello.c
 	@docker build --build-arg PSW_VERSION=$(PSW_VERSION) --build-arg MYST_RELEASE_VERSION=$(MYST_RELEASE_VERSION) -t $(DOCKER_IMG) .
 
 run: build
-	@docker run --rm --device=/dev/sgx:/dev/sgx $(DOCKER_IMG)
+	@docker run --rm --device=/dev/sgx:/dev/sgx --volume /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket $(DOCKER_IMG)
```

Comment:
Adding this --volume makes it possible to do in proc attestation?

Reply:
Yep, this is the volume required. It is also in the job yaml for this sample